doc/sai-ptf/lag_test_plan.md

```diff
@@ -0,0 +1,177 @@
+# SAI Lag Test plan
+- [SAI Lag Test plan](#sai-lag-test-plan)
+  - [Overriew](#overriew)
+    - [Test Topology](#test-topology)
+    - [Testbed](#testbed)
+  - [Scope](#scope)
+- [Basic Configurations SAI API and sample packets](#basic-configurations-sai-api-and-sample-packets)
+  - [Basic Configurations And SAI API](#basic-configurations-and-sai-api)
+  - [Packets](#packets)
+- [Test suites](#test-suites)
+  - [Test suite #1: PortChannel Loadbalanceing](#test-suite-1-portchannel-loadbalanceing)
+  - [Test suite #2: Ingress/Egreee disable](#test-suite-2-ingressegreee-disable)
+  - [Test suite #3: Remove Lag member](#test-suite-3-remove-lag-member)
+  - [ToDO Test suite #4: Scaling test cases](#todo-test-suite-4-scaling-test-cases)
+  - [ToDO Test suite #5: Hash LoadBalancing test cases](#todo-test-suite-5-hash-loadbalancing-test-cases)
+## Overriew
+The purpose of this test plan is to test the LAG/PortChannel function from SAI.
+
+### Test Topology
+For SAI-PTF, it will use a non-topology network structure for the sai testing. 
+
+### Testbed
+Those tests will be run on the testbed structure, the components are:
+* PTF - running in a server that can connect to the target DUT
+* SAI server - running on a dut
+
+*p.s. cause the SAI testing will not depend on any sonic components, then there will be no specific topology(T0 T1 T2) for testing.*
```

Comment:
i do not understand this, why we are spending so much time to setup the t0 testbed then?

Reply:
updated structure and replaced with a more specific statement

*P.S. All the tests are targeted on the T0 scenario.*